src/js/controller/action.js

```diff
@@ -1,16 +1,47 @@
+import { AddtoStorage } from './storage';
+
 export const ArrangeList = (list) => list.map((item, index) => ({ ...item, index }));
 
 export const AddtoList = (description, list) => {
-  const todo = [{ index: 1, description, completed: false }, ...list];
+  const todo = [...list, { index: 1, description, completed: false }];
   return ArrangeList(todo);
 };
 
 export const setCompleted = (id, value, list) => {
   const todos = list.map((todo) => {
-    if (todo.index === id) { return { ...todo, completed: value }; }
+    if (todo.index.toString() === id) { return { ...todo, completed: value }; }
     return todo;
   });
   return todos;
 };
 
-export const removeCompleted = (list) => list.filter((todo) => !todo.completed);
\ No newline at end of file
+export const removeCompleted = (list) => {
```

Comment:
- [ ] This is great that you have implemented the editing functionality requested by the previous reviewer. However, deleting a single item clicking on a trash icon buton is still not addressed yet. Kindly implement that feature too. Take a look at [this video](https://www.youtube.com/watch?v=AcUd-_Yjjqg) to see the feature.